app/interface/deps/usecases/student_deps.py

```
@@ -0,0 +1,29 @@
+) -> StudentAuthenticationUseCase:
+    return StudentAuthenticationUseCase(client=client)
+
+
+def get_student_information_usecase(
+    scraper: LmsScraper = Depends(get_lms_scraper, use_cache=False),
+) -> StudentInformationUseCase:
+    return StudentInformationUseCase(scraper=scraper)
+
+
+def get_student_timetable_usecase(
+    request: StudentTimetableRequest = Body(...),
```

Ruff (0.12.2)

Do not perform function call `Body` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)